app/services/analytics.rb

```diff
@@ -0,0 +1,61 @@
+require 'staccato/adapter/validate'
+require 'staccato/adapter/logger'
+
+class Analytics
+  TRACKER = if Rails.env.test?
+              Staccato.tracker(nil)
+            else
+              Staccato.tracker('UA-48605964-44', nil, ssl: true)
```

Comment:
It can be if we want. I left it here for now since it's not a secret value and since it's not likely to change once we go live.